src/routes/edit/elements-pane.tsx

```diff
@@ -49,14 +49,14 @@ export const ElementsPane = ({isActive}: any) => {
 			<div className={elementTileListStyle}>
 				{
 					Object.values(allComponents)
-					.filter((component: any) =>
-						!component.componentInfo.hideFromElementsPane
-						&& shouldShow(component.componentInfo.keywords))
-					.map((component: any) =>
-						<ElementTile componentObj={component.componentInfo.defaultComponentObj}>
-							<img src={component.componentInfo.image} alt={component.componentInfo.name} />
-							<span className='title'>{component.componentInfo.name}</span>
-						</ElementTile>)
+						.filter((component: any) =>
+							!component.componentInfo.hideFromElementsPane
+							&& shouldShow(component.componentInfo.keywords))
+						.map((component: any, index: number) =>
+							<ElementTile componentObj={component.componentInfo.defaultComponentObj} key={index}>
```

Comment:
it would be better to use a unique property from component instead of index

Reply:
Using component.componentInfo.name, assuming it will always be unique.